digital_comms/ccam.py

```diff
@@ -289,15 +415,29 @@ def _small_cell_capacity(self):
 
     @property
     def capacity_margin(self):
+        """obj: The capacity margin in this postcode sector TODO
+        """
         capacity_margin = self.capacity - self.demand
         return capacity_margin
 
 
 def pairwise(iterable):
     """Return iterable of 2-tuples in a sliding window
 
+    Parameters
+    ----------
+    iterable: list
```

Comment:
I'm not familiar with pairwise - perhaps you can explain it to me?

Reply:
Me neither when I first looked at this. Therefore I've added an example to the docstring:

    Example
    -------
        >>> list(pairwise([1,2,3,4]))
            [(1,2),(2,3),(3,4)]